bioptim/interfaces/SolverOptions.py

```diff
@@ -40,19 +40,30 @@ def set_constraint_tolerance(self, tol: float):
     @abstractmethod
     def set_maximum_iterations(self, num: int):
         """
-         This function set the number of maximal iterations.
+        This function set the number of maximal iterations.
 
-          Parameters
+        Parameters
         ----------
-        tol: int
+        num: int
             Number of iterations
         """
 
+    @abstractmethod
+    def finalize_options(self, solver):
+        """
+        This function return the finalize options structure to launch the optimization
+
```

Comment:
From the comment, it seems that this function return something, but the Return section is not writing present and there is no return type `def finalize_options(self, solver) -> Any:` (Any if the type is unknown at that point)